styles/unsupportedScreen.css

```
@@ -0,0 +1,28 @@
+@media screen and (max-width: 374px) and (min-width: 0px) {
```

💊 제안
이런 경우 min-width를 제거하시는 것이 더 좋을 것 같아요~

```suggestion
@media screen and (max-width: 374px) {
```